### DISCUSSION TITLE
Invio fatture sistema TS

### DISCUSSION THREAD

**Discussion by zpj61** [fattureincloud/api#218] [Q&A] [answered]
Ho cercato nelle API qualcosa riguardante l'invio al sistema TS ma non ho trovato nulla. Mi è sfuggito qualcosa oppure è così?

**Comment by emajo** [comment, marked as answer]
Ciao @zpj61 purtroppo attualmente non esponiamo metodi per invio al sistema tessera sanitaria.

Grazie,
Emanuele.
